src/storage/mongodb/LoggingStorage.ts

```diff
@@ -80,6 +80,7 @@ export default class LoggingStorage {
     // Insert
     if (global.database) {
       await global.database.getCollection<Log>(tenantID, 'logs').insertOne(logMDB);
+      return logMDB._id.toHexString();
```

Comment:
_id already have an ObjectID type. No need to convert it, just return it and reuse. 
It's a hot code path, it should be as light as possible.

Reply:
I tried but it seems to fail without this. Also - this is how I saw id-s being used throughout the code.

Reply:
Read carefully my comment: `_id` already have type defined in the mongodb package types declaration. No need to do an heavy type conversion in a hot code path, just use the type already defined for the `_id` field.

Reply:
I think that the _id should be returned by the insertOne() method, right?

Reply:
> I think that the _id should be returned by the insertOne() method, right?

It's returned also as part of the data structure returned `InsertOneWriteOpResult<WithId<TSchema>>` but it has the same value as the one generated by ObjectId() instantiation in the method.

Reply:
Where is the ObjectId() instantiation in the method?

![image](https://user-images.githubusercontent.com/25266945/108720252-de112180-7520-11eb-9107-0fc367de5faa.png)

Does the MongoDB insertOne() method adds it in the logMDB object?

Reply:
ObjectId() is called inside insertOne() and the object saved is passed as a reference, not value.